### ISSUE TITLE
Prevent error of sonarjs/no-duplicate-string with CSS classes (Tailwind, Foundation, Boostrap...)

### ISSUE THREAD

**Issue by RodrigoTomeES** [SonarSource/eslint-plugin-sonarjs#412]
**I want to request a feature.**

I want that sonarjs/no-duplicate-string doesn't throw erros with Tailiwnd classes.

```js
const Card = () => 
<div class="flex">
  <div class="flex">
    <img .../>
  </div>

  <div class="flex" />
</div>
```

**Comment by arianrhodsandlot** [comment]
Would this option meet your needs?
https://github.com/SonarSource/eslint-plugin-sonarjs/blob/8e3a6835e646c15bfe76f30fc7f5dd37c49579a1/docs/rules/no-duplicate-string.md#L24

**Comment by RodrigoTomeES** [comment]
I think that doesn't meet my need because Tailwind has hundreds or thousands of classes 🙈

**Comment by alexander-kamushkin-sonarsource** [comment]
Thanks a lot - that indeed makes sense, we will look at it.

**Comment by RodrigoTomeES** [comment]
@alexander-kamushkin-sonarsource great thanks! It would be awesome if the implementation can be made for css classes in general because in this case others CSS frameworks will work

**Comment by yassin-kammoun-sonarsource** [comment]
@RodrigoTomeES I analyzed the snippet you provided and the rule doesn't report any issue on it. I also checked the rule implementation and it already makes an exception to all JSX attributes.

Was this a feature suggestion or are you facing an actual false positive from the rule?

**Comment by RodrigoTomeES** [comment]
@yassin-kammoun-sonarsource now I on vacation and I can't provide you an real example but when I will go back to my work I will show you real code that have this problem

**Comment by man-trackunit** [comment]
I have the same issue. Mostly happens when for me with tailwind-merge using an array of classes like this: 
<img width="1018" alt="CleanShot 2023-08-25 at 14 57 36@2x" src="https://github.com/SonarSource/eslint-plugin-sonarjs/assets/79450095/77ba5c59-f4a6-419f-83fd-2d260e19c048">

**Comment by olafurns7** [comment]
To bump, it would be sweet to be able to pass a regex to `ignoreStrings`

**Comment by Wohops** [comment]
This issue has been migrated to Jira. [ESLINTJS-26](https://sonarsource.atlassian.net/browse/ESLINTJS-26)
